# Total planning attempts
max_solutions: 10

# Planning group and link names
arm_group_name: "panda_arm"
eef_name: "hand"
hand_group_name: "hand"
hand_frame: "panda_link8"

# Poses
hand_open_pose: "open"
hand_close_pose: "close"
arm_home_pose: "ready"

# Scene frames
world_frame: "world"
base_frame: "panda_link0"
surface_link: "table"

# Gripper grasp frame transform [x,y,z,r,p,y]
diagonal_frame_transform: [0, 0, 0.1, 2.1855794, .7071, -2.5254162] 
downward_frame_transform: [0, 0, 0.1, 3.142, 0.0, .785]
horizontal_frame_transform: [0, 0, 0.1, 1.571, 0.785, 1.571]
vertical_frame_transform: [0, 0, 0.1, -1.5707963, .7858908, -3.1415927]

place_surface_offset:  .0001 # place offset from surface

# Valid distance range when approaching an object for picking
approach_object_min_dist: 0.1
approach_object_max_dist: 0.15

# Valid height range when lifting an object after pick
lift_object_min_dist: 0.01
lift_object_max_dist: 0.1
